src/workers/types.ts

```diff
@@ -190,16 +193,19 @@ type ProcessInfoBase = {
   name: string;
 };
 
+export type GeneralProcessIssue = 'GET_DEVICE_NAME_ERROR';
 export type ProcessIssue = ProcessInfoBase & {
   action:
     | 'PULL_ERROR'
     | 'RENAME_ERROR'
     | 'RENAME_ERROR'
     | 'DELETE_ERROR'
-    | 'METADATA_READ_ERROR';
+    | 'METADATA_READ_ERROR'
+    | GeneralProcessIssue;
```

Comment:
I think it would be better to have an entirely separate list with the `GENERAL` issues since there is data that is not relevant to a general issue as the `FileSystemKind` and the actions make reference to file systems actions

Reply:
ok, sounds better